### ISSUE TITLE
Custom loss function

### ISSUE THREAD

**Issue by MohammadFakhreddin** [Artelnics/opennn#247]
Hello!,

I am translating my code from keras/tensorflow to opennn. I was wondering how can I write a custom loss function like Keras or TensorFlow?

**Comment by davidge807** [comment]
Hi @MohammadFakhreddin 

Unlike Tensorflow, OpenNN calculates the derivatives of the error functions analytically, so it is much more complicated to implement a loss function of your own.
